x-pack/plugin/security/src/main/java/org/elasticsearch/xpack/security/authc/RealmsAuthenticator.java

```diff
@@ -259,7 +259,7 @@ private static void consumeNullUser(
         messages.forEach((realm, tuple) -> {
             final String message = tuple.v1();
             final String cause = tuple.v2() == null ? "" : " (Caused by " + tuple.v2() + ")";
-            logger.warn("Authentication to realm {} failed - {}{}", realm.name(), message, cause);
+            logger.debug("Authentication to realm {} failed - {}{}", realm.name(), message, cause);
```

Comment:
this change might break an internal dashboard and we might want to leave this at warn until we can replace with it something better.